src/styles/themes/theme-leedsPHR/base/_base.scss

```diff
@@ -4,5 +4,5 @@
 }
 
 .main {
-  border-top: 6px solid #d3d3d3;
+  //border-top: 6px solid #d3d3d3;
```

Comment:
@BogdanScherban I am not sure we want to delete this part

Reply:
It is necessary, because in opposite case we will have clearance between Topbar and Content of the page.
I checked it